en/docs/api-gateway/observability/metrics/viewing-metrics-in-grafana.md

```
@@ -0,0 +1,140 @@
+# Viewing Metrics in Grafana
+
+Once you've started the gateway with the metrics profile, follow these steps to view component metrics:
+
+### Step 1: Access Grafana
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)